scripts/DeployAngleSideChainMultiBridge.s.sol

```diff
@@ -0,0 +1,144 @@
+        } else {
+            expectedAddress = address(0xAaaaAA19ddd491648Db2a31C32f2B2792DeC5015);
+            // TODO compute the expected address once one of the address has been deployed
+        }
+
+        TokenSideChainMultiBridge angleImpl = new TokenSideChainMultiBridge();
+        console.log("TokenSideChainMultiBridge Implementation deployed at", address(angleImpl));
+
+        ImmutableCreate2Factory create2Factory = ImmutableCreate2Factory(IMMUTABLE_CREATE2_FACTORY_ADDRESS);
+        address computedAddress = create2Factory.findCreate2Address(salt, initCode);
+        console.log("TokenSideChainMultiBridge Proxy Supposed to deploy: %s", computedAddress);
+
+        require(computedAddress == expectedAddress, "Computed address does not match expected address");
+
+        TokenSideChainMultiBridge angleProxy = TokenSideChainMultiBridge(create2Factory.safeCreate2(salt, initCode));
+        TransparentUpgradeableProxy(payable(address(angleProxy))).upgradeTo(address(angleImpl));
+        TransparentUpgradeableProxy(payable(address(angleProxy))).changeAdmin(proxyAdmin);
+        console.log("TokenSideChainMultiBridge Proxy deployed at", address(angleProxy));
+
+        LayerZeroBridgeTokenERC20 lzImpl = new LayerZeroBridgeTokenERC20();
+        console.log("LayerZeroBridgeTokenERC20 Implementation deployed at", address(lzImpl));
+        LayerZeroBridgeTokenERC20 lzProxy = LayerZeroBridgeTokenERC20(
+            address(
+                _deployUpgradeable(
+                    proxyAdmin,
+                    address(lzImpl),
+                    abi.encodeWithSelector(
+                        LayerZeroBridgeTokenERC20.initialize.selector,
+                        string.concat("LayerZero Bridge ", symbol),
+                        string.concat("LZ-", symbol),
+                        lzEndpoint,
+                        coreBorrow,
+                        address(angleProxy),
+                        0
+                    )
+                )
+            )
+        );
+        console.log("LayerZeroBridgeTokenERC20 Proxy deployed at", address(lzProxy));
+
+        angleProxy.initialize(
+            symbol,
+            symbol,
+            ICoreBorrow(coreBorrow),
+            address(lzProxy),
+            totalLimit,
+            hourlyLimit,
+            0,
+            false,
+            chainTotalHourlyLimit
+        );
+
+        if (mock) {
+            (uint256[] memory chainIds, address[] memory contracts) = _getConnectedChains("ANGLE");
+
+            // Set trusted remote from current chain
+            for (uint256 i = 0; i < contracts.length; i++) {
+                if (chainIds[i] == chainId) {
+                    continue;
+                }
+
+                lzProxy.setTrustedRemote(_getLZChainId(chainIds[i]), abi.encodePacked(contracts[i], address(lzProxy)));
+            }
+
+            // add real governor
+            if (vm.envBool("FINALIZE")) {
+                address realGovernor = vm.envOr(
```

Comment:
If env is set it will always take the env one? 
Can we do the inverse, by default the sdk one and if none the .env